### PR TITLE
Implemented Merge Sort

### PR DESCRIPTION
**Pull request by jvmolu**
Anmol Singhal
11940140
anmolsinghal@iitbhilai.ac.in

### CODE REVIEW

File: Sorting/merge.cpp [hunk 1]
```diff
@@ -2,9 +2,74 @@
...
- * 
+ *
  * Inputs: null
  */
-void Sort::MergeSort(){
-    std::cout<<"[TODO @ Sorting/merge.cpp] MergeSort() needs to be implemented.\n";
-}
\ No newline at end of file
+ std::vector<int> arr;
+ void Merge(int low, int mid, int high)
```
Comment (shashwatj07) [inline Sorting/merge.cpp:10]: Good work, but you need to do this in a separate PR. This is not type-1. Refer README.
Revert this change.

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
One PR must have only one commit. Please squash all commits and force push branches. Also, keep in mind that one PR must have changes of a single type.
Please consider submitting PRs type by type, from 1 to 4. Refer README.md for more.

**Comment by jvmolu** [comment]
@shashwatj07  i have squashed the commits and revert the type - 1 changes

**Comment by shashwatj07** [comment]
> @shashwatj07 i have squashed the commits and revert the type - 1 changes

Nice. We'll review this later. Once you complete types 1 to 3.
